pages/linux/sfill.md

```diff
@@ -0,0 +1,20 @@
+# sfill
+
+> Secure overwrite free space and inodes of the partition where the specified directory resides.
+> More information: <http://manpages.ubuntu.com/manpages/xenial/man1/sfill.1.html>.
+
+- Overwrite free space and inodes of an USB with 38 writes (slow but secure):
+
+`sfill {{/media/user/mounted_usb_directory}}`
```

Comment:
```suggestion
`sfill {{/path/to/mounted_usb_directory}}`
```

Reply:
Hi @bl-ue, 
thanks for the suggestion, please consider this generic link may a bit more dangerous because if an inexperienced user try to use this command with the wrong folder (e.g. /home/user/my_custom_usb_folder/ where the USB is not currently mounted) the command will **be executed on the partition where the specified directory resides** which is the main one, during the process `sfill` creates a file as large as the free space available and some distro will slow down or even freeze as long as the process is not completed.

Specify "/media/user/mounted_usb_directory" I believe is bit less risky and more intuitive because:

- it is the default path of mounted USB for many (maybe all) distros
- when the USB is not mounted the folder does not exist 

Given this, if you still prefer the `/path/to/mounted_usb_directory` I will change it

Reply:
@mkcn You raise some good points.  We normally like to make it like an English phrase `e.g. path to file`, but I'm sure we can make an exception here.  Maybe @sbrl will have some suggestions.

Reply:
I'm going with @bl-ue 's change, you can mount USBs anywhere so `path to mounted_usb_dir` is better imo. Not every system uses `/media/${USER}/${MountPoint}`, that's more with Desktop Environments and also as said before, not everytime the same on every system.

Reply:
I would suggest that consistency would be the best option here. Some systems mount to `/mnt` by default, while others prefer `/media/username`. So either `/absolute/path/to/mount_point` or `path/to/mount_mount` depending on whether an absolute path is required or not.